README.md

```diff
@@ -209,6 +209,29 @@ trie.put('key', 'value', () => {
 
 ```
 
+## Examples (webpack.config.js)
+
+To bundle with webpack instead of browserify, resolve the `fs` module with the `graceful-fs` module.
+
+```js
+const path = require('path')
+
+module.exports = {
+  entry: './src/dat.js',
+  target: 'web',
+  resolve: {
+    alias: {
+      fs: 'graceful-fs'
+    }
+  },
+  output: {
+    filename: 'dat.js',
```

Comment:
Would this be then entire bundled project, or just the portion referenced by the entry file? Is the point to have multiple bundle files?

Reply:
Just one bundle file please. I went through a couple of different names for testing.

```suggestion
    filename: 'bundle.js',
```
To match https://webpack.js.org/#bundle-it